webapp/app/locales/en/translations.js

```diff
@@ -230,8 +222,8 @@ export default {
       rollbacked_label: 'Rollbacked',
       file_label: 'File:',
       details_label: 'Details:',
-      explanation_label: 'Why did this happened?',
-      rollback_confirm: 'Are you sure you want to rollback this activity? This activity cannot be rollbacked.',
+      explanation_label: 'Why did this happen?',
+      rollback_confirm: 'Are you sure you want to rollback this activity? You cannot undo a rollback.',
```

Comment:
Does this make sense? I thought the original string was a somewhat confusing 😅 .